doc/examples/dask_xgboost/dask_xgboost.py

```diff
@@ -0,0 +1,321 @@
+# flake8: noqa: E501
+"""
+XGBoost-Ray with Dask
+======================
+
+This notebook includes an example workflow using
+`XGBoost-Ray <https://docs.ray.io/en/latest/xgboost-ray.html>`_ and
+`Dask <https://docs.dask.org/en/latest/>`_ for distributed model training,
+hyperparameter optimization, and prediction.
+"""
+
+###############################################################################
```

Comment:
```suggestion
"""

# If you are moving this file, please also update the path in
# release/golden_notebook_tests/workloads/dask_xgboost_test.py

###############################################################################
```

Reply:
I would prefer to move this somewhere else (maybe the README?) as this is not relevant to the users who are the primary readers of the docs/script.

Reply:
Yeah, I wasn't 100% sure where to put this in myself. I can see the case for README, but on the other hand, I feel information like this should be directly in the file it concerns. That being said, having it anywhere is better than nowhere!